src/commands/test/lib/wizard/uitest.ts

```diff
@@ -39,7 +38,7 @@ export default class RunUitestWizardTestCommand extends AppCommand {
   }
 
   private async promptMergeXml(): Promise<boolean> {
-    const questions: Questions = [
+    const questions = [
```

Comment:
That's not a great change. Why remove the type here?

Reply:
The types are outdated. The `Questions` type no longer exist, and `Question` doesn't include `choices` so we can't make it a `Question[]` either.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inquirer/index.d.ts#L247

https://github.com/SBoudrias/Inquirer.js#questions

Reply:
So strange :/